### ISSUE TITLE
Plot information is missing

### ISSUE THREAD

**Issue by marieALaporte** [plantbreeding/API#548]
As discussed during the Agronomy WG, it would be great to add explicitly plot information (plot length, plot width) to an endpoint.

**Comment by cpommier** [comment]
Right. But there are many information of that type that could be added. Using the environment parameter approach as in [MIAPPE](https://github.com/MIAPPE/MIAPPE/blob/master/MIAPPE_Checklist-Data-Model-v1.1/MIAPPE_Checklist-Data-Model-v1.1.pdf) (see parameter list in the anexe, there is plot size) could be a way to solve this and futur request.

**Comment by daveneti** [comment]
I am working on an experimental design extension in case anyone wishes to discuss.

**Comment by BrapiCoordinatorSelby** [comment]
@daveneti If no one has an opportunity to talk with you directly before your deadlines, here is the notes document for where we left off in the experimental design conversation. 
https://docs.google.com/document/d/1ITSaFpRqn01_69YxR2vG80gkXnWo5OSo2kiL7KEyP2M/edit
Entries for June 22, Aug 3, and Aug 17 specifically.

**Comment by ghost** [comment]
Add "Experimental Unit" concept to BrAPI?

Predefined sets of levels and orders is too ridged for species agnostic applications. Specifying the observation unit that defines the spatial layout of the experiment, such as the unit into which germplasm is randomized, is helpful. At BI we are calling this the experimental unit, (often equivalent to plot in row crops). The experimental unit is the highest level of observation within an experiment and all other levels of observation are its children in the order hierarchy (aka sub-observation units). Specifying experimental unit, allows the order of the observation hierarchy to be autogenerated based on the relationship between the experimental unit and sub-observation unit independent of predefine values. Thus auto-establishing the data relationships needed to aggregate summary stats.
